### DISCUSSION TITLE
Macos Mac book pro MAX + UTM + Ubunut 22.04 + 2 monitor =>error from start

### DISCUSSION THREAD

**Discussion by Esysteme** [utmapp/UTM#5189] [UTM Help (macOS)] [answered]
Questions?

Any possibility to start Ubuntu (or any other Gnu/Linux) with at least 2 monitors ?

* UTM Version: 4,1
* macOS Version:  13.2.1
* Mac Chip (Intel, M1, ...):  M2 MAX
* Qemu : 7.2

**Crash log**  
QEMU left with an error : qemu-aarch64-softmmu: -device virtio-ramfb: duplicate fw_cfg file name: vgaroms/vgabios-ramfb.bin

**Comment by rkoskulics** [comment]
I have an identical issue.  Though I don't seem to be able to generate debug logs; I am running a 2 display setup and I cannot get passed the installation.  That goes fine, then on reboot hangs indefinitely at a black screen.

**Comment by osy** [comment, marked as answer]
This is poorly documented but you have to choose a different graphics hardware for your second monitor. “secondary-vga” for example should work.

**Reply by bendavis78** [reply]
What if we have two external monitors? Is it not possible to use the same graphics type? I noticed it doesn't allow multiple hardware accelerated displays, which is a bummer. But I should at least be able to add multiple non-accelerated displays, right?
